apps/frontend/src/pages/auth/components/auth-layout/auth-layout.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { Logo } from "~/libs/components/components.js";
+import { LogoSize } from "~/libs/enums/enums.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	children: React.ReactNode;
+};
+
+const AuthLayout = ({ children }: Properties): React.JSX.Element => {
```

Comment:
To be consistent with other components
```suggestion
const AuthLayout: React.FC = ({ children }: Properties) => {
```